E.Deezer/Endpoint/BrowseEndpoint.cs

```diff
@@ -69,10 +69,12 @@ public async Task<IAlbum> GetAlbumById(uint albumId)
             };
 
             // Will throw if Deezer error
-            var response = await iClient.GetDeezerObject<AlbumObjectResponse>("album/{id}", p)
+            var response = await iClient.GetPlain<Album>("album/{id}", p)
```

Comment:
Not sure if this is correct. I'll need to check this out....

Reply:
Yes, You were wright this was a mistake from me. I pushed a new commit which revert it to the original version and added a few tests just to make sure it won't happen again (**78.05%** code coverage).